src/components/panels/RemoteAppsPanel/RemoteAppsPanel.tsx

```diff
@@ -0,0 +1,44 @@
+import useModelStatus from "hooks/useModelStatus";
+import { ReactElement } from "react";
+
+import { generatePanelTableRows } from "../shared";
+
+import "../_panels.scss";
+
+type Props = {
+  entity: string;
+};
+
+export default function RemoteAppsPanel({
+  entity: appId,
+}: Props): ReactElement {
+  // Get model status info
+  const modelStatusData = useModelStatus();
+
+  const remoteApp = modelStatusData["remote-applications"][appId];
```

Comment:
I believe that `remote-applications` can also be undefined so this should be a bit more resilient around if this appId can not be found.